### ISSUE TITLE
The Import Time Take So Long

### ISSUE THREAD

**Issue by trnkobayashi** [themoeway/yomitan#1313]
I tried to import [JPDB Frequency](https://drive.google.com/drive/folders/1g1drkFzokc8KNpsPHoRmDJ4OtMTWFuXi) and [Jitendex](https://drive.google.com/drive/folders/1RXSkwOLLcSEIptGpoiDY2eaYAa-Ohu6x) but It was very slow for some reason, about 7% for 3 hours (from 41% to 48%)

![ảnh](https://github.com/user-attachments/assets/bacc753b-2f0b-4ac5-b09a-31d1ec64ae09)

Firefox 129.0.1

Yomitan 24.7.31.0

I'm just new to this and haven't change any settings

**Comment by MarvNC** [comment]
This is a known problem due to Firefox's implementation of indexeddb, Chromium browsers will import much faster.

**Comment by Kuuuube** [comment]
Firefox's implementation is fine it just isn't optimized as well in most cases.

Unless you're on a low spec device it really shouldn't take that long to import. Maybe it's hitting some limit and throttling down hard. Try restarting the browser and importing again. If you interrupted the import of any dicts you will also need to delete them first before re-importing since they will be partially imported and still show as imported.

I just did a test importing JPDB dict on firefox and chromium:
```
0:48 Firefox
1:06 Chromium
```
So close to a minute for both. Not an enormous difference.

Jitendex will take longer for sure but it should never be taking 3 hours. Probably around 10-30 mins max.

**Comment by trnkobayashi** [comment]
> This is a known problem due to Firefox's implementation of indexeddb, Chromium browsers will import much faster.

You're right, my chrome import it really fast, maybe I will use it instead

**Comment by trnkobayashi** [comment]
> Try restarting the browser and importing again.

Well, it actually works. The problem might have happened because I did something in about:config

**Comment by Casheeew** [comment]
Closing as completed
